sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -103,7 +104,7 @@ def test_custom_coder(self):
 
     self.check_coder(CustomCoder(), 1, -10, 5)
     self.check_coder(coders.TupleCoder((CustomCoder(), coders.BytesCoder())),
-                     (1, 'a'), (-10, 'b'), (5, 'c'))
+                     (1, b'a'), (-10, b'b'), (5, b'c'))
 
```

Comment:
Probably not critical, but looks like 'a' is not replaced with b'a' here - are these changes done by some tool or manually?

Reply:
I guess this is aimed at the 'a' in line 109? 
The marking of the strings as bytes literals is done manually. I've only marked strings as bytes literals when it's clear that they're meant to represent bytes (when testing BytesCoder, when the content of the string are clearly bytes, ...).  When a string is not marked, it represents `str` on both versions, which seems ok for the 'a' at line 109 for example.